### PR TITLE
MEB-98: Add logging of config values and git commit on init

### PR DESCRIPTION
**Pull request by paramsingh**
Also, removed the `runserver` command from the dockerfile and moved it into `docker-compose.yml` for consistency with LB and MsB etc.

### CODE REVIEW

**Comment by mayhem** [comment]
@brainzbot please test this

File: docker/docker-compose.dev.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ services:
     build:
       context: ..
       dockerfile: ./docker/Dockerfile.dev
+    command: python /code/manage.py runserver -h 0.0.0.0 -p 80
```
Comment (mayhem) [inline docker/docker-compose.dev.yml]: curious, why did you move this?

File: metabrainz/utils.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 from urllib.parse import urlparse, parse_qsl, urlunparse, urlencode
-import string
+
 import random
+import string
```
Comment (mayhem) [inline metabrainz/utils.py]: Is this needed?

**Comment by mayhem** [comment]
@brainzbot please add this to the whitelist

**Comment by mayhem** [comment]
@brainzbot, add to whitelist

**Comment by mayhem** [comment]
@brainzbot, you're useless twat.

File: docker/docker-compose.dev.yml (hunk 1 above)
Comment (paramsingh) [reply docker/docker-compose.dev.yml]: @mayhem, I was looking at the Dockerfile to see why debug messages didn't seem to be getting printed, so I tried moving the command here. Anyways, doing this in the compose file looked better to me (and it is consistent with how we do it in LB and MsB etc.)

File: metabrainz/utils.py (hunk 2 above)
Comment (paramsingh) [reply metabrainz/utils.py]: I moved the imports around to make them alphabetical. The import is  used in a the [generate_string](https://github.com/metabrainz/metabrainz.org/blob/81fc69a531080928f41657f6a755f4827ddd4820/metabrainz/utils.py#L14) function.
